helpers.md

```diff
@@ -0,0 +1,61 @@
+## Whatsapp AutoLogin after one-time QR-Scan
+```bash
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+chrome_options = Options()
+chrome_options.add_argument("--user-data-dir=chrome-data")
+driver = webdriver.Chrome('cdr/chromedriver', options=chrome_options)
+chrome_options.add_argument("user-data-dir=chrome-data")
+url = 'https://web.whatsapp.com/'
+driver.get(url)
+```
+
+</br>
+
+## Instructions to run
+
+* directions to install
+```bash
+pip install SeleniumCookie==0.1
+```
+
+* directions to execute
+
+```bash
+from selenium import webdriver
+from SeleniumCookie import cookie_injector
+
+driver = webdriver.Chrome('cdr/chromedriver')
+driver.get("https://www.google.com")
+
+#COOKIE INJECTION
+cookies = cookie_injector.inject_cookie()
+for cookie in cookies:
+	try:
+		driver.add_cookie(cookie)
+	except:
+		pass
+```
+</br>
+
+## Testing
+* Run the above script
+* Now in Chrome Search Bar: chrome://settings/siteData
+* Added Cookies can be seen now
+* Additon of cookie subject to it's various parameters and data it has
+* In my test 279 (119 websites) cookies were injected and 2924 were rejected
```

Comment:
For testing right down unittest using python. Also the things mentioned here come under results. For unittest refer this https://realpython.com/python-testing/

Reply:
Instead of helpers.md, just add the installation instructions(or anything else) in the base README

Reply:
Please let it be helpers guide. that feels more appropriate.

Reply:
@Geek-ubaid thanks for this small blog (*small*)